src/states/node.rs

```diff
@@ -103,6 +104,8 @@ pub struct Node {
     tick_timer_token: u64,
     timer: Timer,
     tunnels: Tunnels,
+    /// Key is (dst, sections), value is BTreeSet<src>
+    out_of_seq_own_merge: LruCache<(Prefix<XorName>, SectionMap), BTreeSet<Prefix<XorName>>>,
```

Comment:
Why is this _indexed_ by the section map? Won't that be different even for a matching pair of `OwnSectionMerge` messages?